package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "bat-usermodel",
+  "name": "@brave/bat-usermodel",
```

Comment:
@evq - shouldn't that be `@brave-intl` ??? just checking.

Reply:
I don't think we have a `brave-intl` npm org

Reply:
but it's not taken and we might as well use it - will push a change